protos/bottle/notification/organization/v1/created.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package bottle.notification.organization.v1;
+
+import "bottle/account/v1/organization.proto";
+
+message Created {
+  bottle.account.v1.Organization organization = 1;
```

Comment:
Alright, now I'm curious the difference between `notification` and `events`. To me they are the same. Help desk would just listen for these events and mirror. Our notification service would also listen to the event to send a welcome email, etc.

Reply:
Events should replace notifications. We didn't spend the time we should have thinking this through when rolling out Bullhorn.

We can depreciate and remove the notifications once we change everything over to use the events.